aicsimageio/readers/reader.py

```diff
@@ -204,6 +204,9 @@ def get_channel_names(self, scene: int = 0) -> Optional[List[str]]:
         channel_dim_size = self.dask_data.shape[channel_index]
         return [str(i) for i in range(channel_dim_size)]
 
+    def get_ome_metadata(self):
+        return NotImplementedError()
```

Comment:
Is this supposed to return, or raise?

Reply:
I prefer it to `raise`. Unlike the other methods in the base reader, this one is very much tied to "have we written metadata transforms for each metadata type" and we haven't / it isn't possible in some situations. `DefaultReader` for example will never have a `get_ome_metadata` function imo.

Reply:
Ok but how does a user know what to do with it?  I feel you might as well just return None.  At least then i can say `ome = get_ome_metadata();  if ome then...`

Reply:
returning an Error object just doesn't look right here.

Reply:
OH! I thought it was `raise NotImplementedError()` yea. returning an error is incorrect here. RAISING and error is what we want